docs/collaboration.md

```diff
@@ -0,0 +1,68 @@
+# Collaborative Publishing with Posit Publisher
+
+This workflow guide explains how to collaborate with others on deployed content.
+
+## Configuration and Deployment files
+
+Posit Publisher creates and utilizes two `.toml` files for deploying content.
+Both are inside of the `.posit/publish` directory alongside your entrypoint
+file.
+
+- The Configuration file describes how the content will be deployed to Posit
+  Connect. It can be edited by hand and by interacting with the Posit
+  Publisher sidebar. It is named using the content's title and a unique
+  identifier, for example, `content-title-061E.toml`. See the
+  [Configuration file reference](./configuration.md) for a full list of
+  settings.
+- The Deployment file details where the content was deployed and its state.
+  Deployment files are automatically generated by Posit Publisher in the
+  `deployment` directory and updated during deploys. They are not meant to be
+  edited. Posit Publisher will take care of this file for you so you do not need
+  to think about it aside from the collaboration tips below.
+
+Utilizing both of these files, alongside a credential and collaborator access
+on Posit Connect, you and your collaborators can update and configure the same
+content. To grant others collaborator access see
```

Comment:
```suggestion
Utilizing both of these files, alongside a credential and collaborator access
on Posit Connect, you and your collaborators can update and configure the same
content. To grant others collaborator access see
```

This is very minor, please feel free to ignore, but I found `alongside a credential and collaborator access` a bit odd here. It's of course true you would need a credential and collaborator access, but I'm not sure calling that out here is all that helpful? Is there something you're trying to call out here besides something like basic requirements? These both might be low-level enough that we don't need to list them (like we don't list things like a network connection to the connect server, a Python or R binary, etc.)

Reply:
>  Is there something you're trying to call out here besides something like basic requirements? These both might be low-level enough that we don't need to list them (like we don't list things like a network connection to the connect server, a Python or R binary, etc

That is fair. I wanted to remind users that collaborator access should be granted when giving someone the files to collaborate (through Git or another method). I know frequently when collaborating on a deployment I have forgotten to add collaborator access until the other person let me know the deploy failed, so figured it was worth the callout here.

Reply:
After looking at this for a minute I removed the "alongside a credential and collaborator access" bit, and just add a bit before the documentation link

> Utilizing both of these files you and your collaborators can update and
configure the same content. To grant others collaborator access to content see
[Posit Connect User Guide - Access list settings](https://docs.posit.co/connect/user/content-settings/#set-collaborators).

So there is still a link to follow if you hadn't collaborated before, or need the reminder, but it isn't as repetitive or calling out basic things.